_episodes/25.md

```diff
@@ -0,0 +1,217 @@
+     time: 4265
+---
+
+## Guests
+ * Ayush Chauhan, Data Platform Engineer at [Zomato](https://www.zomato.com/who-we-are)
+ ([Ayush Chauhan](https://www.linkedin.com/in/ayush-chauhan/)).
+ * Gunnar Morling, Lead of Debezium and Open source software engineer at [Red Hat](https://www.redhat.com)
+ ([@gunnarmorling](https://twitter.com/gunnarmorling)).
+ * Ashhar Hasan, Software Engineer at [Starburst](https://starburst.io/)
+ ([@hashhar](https://twitter.com/hashhar)).
+ 
+## Release 361
+
+Official announcement items from Martin:
+
+* Support for OAuth2/OIDC opaque access tokens
+* Aggregation pushdown for Pinot
+* Better performance for Parquet files with column indexes
+* Support for reading fields as JSON values in Elasticsearch
+
+Manfred's additional notes:
+
+ * Predicate pushdown in Cassandra
+ * Metadata cache size limitation in a few connectors
+ * Lots of improvements for Hive view support
+ * Glue table statistics improvements
+ 
+More info at <https://trino.io/docs/current/release/release-361.html>.
+
+## Concept of the week: Change Data Capture
+
+If you know Trino, you know it allows for flexible architectures that include 
+many systems with varying use cases they support. We've come to accept this 
+potpourri of systems as a general modus operandi for most businesses. 
+
+Many times the data gets copied to different systems to accomplish varying use 
+cases from performance and data warehousing to merge cross cutting data into a 
+single store. When copying data between systems, how do these systems stay in 
+sync? It's a critical need especially for Trino to know that the state across 
+the data sources we query are valid. 
+
+To answer this, we can use the concept of Change Data Capture (CDC). CDC is a 
+powerful concept that considers a data source(s), called a system of record(s), 
+that store the true state of a system. The system of records are monitored for
+changes, and upon detecting changes, the CDC system propogates changes to a 
+number of target systems.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/25/cdc.png"/><br/>
+Change Data Capture: <a href="https://medium.com/event-driven-utopia/a-gentle-introduction-to-event-driven-change-data-capture-683297625f9b">Source</a>.
+</p>
+
+### Debezium for CDC
+
+One implemention of CDC that has grown tremendously in popularity since its 
+inception is called Debezium. According to <https://debezium.io>:
+
+> Debezium is an open-source distributed platform for change data capture. Start
+> it up, point it at your databases, and your apps can start responding to all 
+> of the inserts, updates, and deletes that other apps commit to your databases.
+> Debezium is durable and fast, so your apps can respond quickly and never miss
+> an event, even when things go wrong.
+
+The common way Debezium is deployed in the wild is using Kafka Connect as the
+CDC servers by defining the Debezium source connectors. You can then use the 
```

Comment:
Also .. reword this sentence to be clearer .. I think Kafka Connect is only used for targets .. not to pull data from source